tests/bgp/test_bgpmon.py

```diff
@@ -143,10 +109,12 @@ def build_syn_pkt(local_addr, peer_addr):
 
 
 def test_bgpmon(dut_with_default_route, localhost, enum_rand_one_frontend_asic_index,
-                common_setup_teardown, set_timeout_for_bgpmon, ptfadapter, ptfhost):
+                common_setup_teardown, set_timeout_for_bgpmon, ptfadapter, ptfhost, tbinfo):
     """
     Add a bgp monitor on ptf and verify that DUT is attempting to establish connection to it
     """
+    if tbinfo['topo']['type'] == 't2':
```

Comment:
iso of skipping here, 
```suggestion
    if tbinfo['topo']['type'] == 't2':
```can we add conditional marker to skip these test?